.github/workflows/ci.yml

```diff
@@ -167,3 +167,26 @@ jobs:
         with:
           command: clippy
           args: --no-deps
+
+  release-check:
+    name: Build in release mode
+    runs-on: ubuntu-latest
+    env:
+      RUSTFLAGS: "-D warnings"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: Swatinem/rust-cache@v2
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install libpq-dev
+      - name: Cargo check (debug)
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+          args: --tests
```

Comment:
Could we remove this step of running debug check with `--tests`, maybe making it a bit faster?

Reply:
I think we can, running the tests covers it already. 👍

Reply:
Fixed in 344e95da865554c17a8c66363f9eb8bf71458429.